.github/actions/version-matrix/package.json

```diff
@@ -5,6 +5,9 @@
 		"python:normal": "bun src/matrices/python/normal.ts",
 		"python:playwright": "bun src/matrices/python/playwright.ts",
 		"python:selenium": "bun src/matrices/python/selenium.ts",
+		"node:normal": "bun src/matrices/node/normal.ts",
+		"node:puppeteer": "bun src/matrices/node/puppeteer.ts",
+		"node:playwright": "bun src/matrices/node/playwright.ts",
```

Comment:
If we started using Node 23 in CI, would it mean we could just use plain old node for this?

Reply:
Possibly yes

Reply:
Can we make an issue for that? Not that I'm a bun hater or anything, but it's one more piece of tech we depend upon...